code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -36,7 +36,7 @@
 
 /obj/item/weapon/melee/baton/proc/deductcharge(var/chrgdeductamt)
 	if(bcell)
-		if(bcell.charge < (hitcost+chrgdeductamt)) // If after the deduction the baton doesn't have enough charge for a stun hit it turns off.
```

Comment:
Wouldn't using `<=` instead of `<` here turn the baton off if the baton has exactly enough charge for one hit?

Reply:
After looking at it, Citinited is right, this should shut the baton off if it has just enough charge.

Reply:
After looking at it, Citinited is right, This should shut the baton off if it has just enough charge.Please fix this